manim/utils/config_utils.py

```diff
@@ -122,6 +121,27 @@ def _parse_file_writer_config(config_parser, args):
         [fw_config["save_last_frame"], fw_config["from_animation_number"]]
     )
 
+    # Read in the log level
+    quiet = getattr(args, "quiet")
+    if not quiet and default.getboolean("quiet", None) is not None:
+        quiet = default.getboolean("quiet")
```

Comment:
Yes absolutely `quiet` is just a shortcut for `verbose CRITICAL`. I thought it could be easier for a user that wants almost no output to remember `quiet` than `verbose CRITICAL`. However it does add an additional option so if it is confusing I'll remove it.